docker/ubuntu_18.04-dev.dockerfile

```diff
@@ -125,9 +125,9 @@ ARG build_pyne=YES
 # Build/Install PyNE
 RUN if [ "$build_pyne" = "YES" ]; then \
         cd $HOME/opt \
-        && git clone https://github.com/cnerg/pyne.git \
+        && git clone https://github.com/pyne/pyne.git \
```

Comment:
These are important changes that we missed!!  Can you make a separate PR with these corrections - and any similar ones that remove reference to CNERG or specific repo branches?

Reply:
The separate PR about dockerfile see https://github.com/pyne/pyne/pull/1172. I searched the code with `cnerg/pyne` and `pymoab_clean`, not any outdated ones found.